src/docs/development/ui/widgets-intro.md

```diff
@@ -711,13 +728,16 @@ class ShoppingListItem extends StatelessWidget {
         backgroundColor: _getColor(context),
         child: Text(product.name[0]),
       ),
-      title: Text(product.name, style: _getTextStyle(context)),
+      title: Text(
+        product.name,
+        style: _getTextStyle(context),
+      ),
     );
   }
 }
 
 class ShoppingList extends StatefulWidget {
-  ShoppingList({Key? key, required this.products}) : super(key: key);
```

Comment:
I'm curious as to why the args got re-ordered =)

Reply:
Manual change. I like it when the key is last, since it's the least interesting. 😄

Reply:
sweet, sgtm